src/pages/index.tsx

```diff
@@ -19,10 +17,27 @@ function HomepageHeader() {
           <div className="hero-content">
             <div className="hero-content-inner">
               <h1 className="hero-title">The Free Software Media System</h1>
-              <p className="hero-text">Jellyfin is the volunteer-built media solution that puts <em>you</em> in control of your media. Stream to any device from your own server, with no strings attached. Your media, your server, your way.</p>
-              <a href="https://demo.jellyfin.org/stable" className="button button--secondary button--outline">See it in Action</a>
-              <a href="/downloads" className="button button button--primary margin-horiz--md">Download Now</a>
-              <a href="" className="button button--secondary button--outline scroll">Learn More</a>
+              <p className="hero-text">
+                Jellyfin is the volunteer-built media solution that puts{' '}
```

Comment:
What's with the weird space in brackets?

Reply:
> This is used to put an explicit space in a text block, since leading and trailing spaces are ignored at compile/transformation time when there is another tag.

https://stackoverflow.com/a/38547635/7617396